modules/tools/migration/src/main/java/org/wso2/mb/migration/DBConnector.java

```diff
@@ -225,6 +267,203 @@ void writeMessageRouter(String routerName, String routerData) throws SQLExceptio
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                String queueName = resultSet.getString(QUEUE_NAME);
+                if(queueNameHasCapitals(queueName)) {
+                    String newQueueName = queueName.toLowerCase();
+
+                    PreparedStatement updateStatement = conn.prepareStatement(UPDATE_MB_QUEUE_MAPPING);
+                    updateStatement.setString(1, newQueueName);
+                    updateStatement.setString(2, queueName);
+                    updateStatement.executeUpdate();
+                }
+            }
+
+        } catch (SQLException e) {
+            e.printStackTrace();
+        } finally {
+            conn.close();
+        }
+    }
+
+    /**
+     * Make queue names in  MB_SLOT_MESSAGE_ID table all simple
+     *
+     * @throws SQLException in case of executing update
+     */
+    void updateQueueNamesInSlotMessageIds() throws SQLException {
+        getConnection();
+        try {
+            PreparedStatement preparedStatement = conn.prepareStatement(GET_MB_SLOT_MESSAGE_IDS);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                String queueName = resultSet.getString(QUEUE_NAME);
+                if(queueNameHasCapitals(queueName)) {
+                    String newQueueName = queueName.toLowerCase();
+
+                    PreparedStatement updateStatement = conn.prepareStatement(UPDATE_MB_SLOT_MESSAGE_ID);
+                    updateStatement.setString(1, newQueueName);
+                    updateStatement.setString(2, queueName);
+                    updateStatement.executeUpdate();
+                }
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        } finally {
+            conn.close();
+        }
+    }
+
+    /**
+     * Make all queue names in MB_QUEUE_TO_LAST_ASSIGNED_ID table all simple
+     *
+     * @throws SQLException in case of executing update
+     */
+    void updateQueueNamesInQueueToLastAssignedIds() throws SQLException {
+        getConnection();
+        try {
+            PreparedStatement preparedStatement = conn.prepareStatement(GET_MB_QUEUE_TO_LAST_ASSIGNED_IDS);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                String queueName = resultSet.getString(QUEUE_NAME);
+                if(queueNameHasCapitals(queueName)) {
+                    String newQueueName = queueName.toLowerCase();
+
+                    PreparedStatement updateStatement = conn.prepareStatement(UPDATE_MB_QUEUE_TO_LAST_ASSIGNED_ID);
+                    updateStatement.setString(1, newQueueName);
+                    updateStatement.setString(2, queueName);
+                    updateStatement.executeUpdate();
+                }
+
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
Change this.

Reply:
Hi Pamod,
This was fixed with following pr - #511

Reply:
Hi Pamod,
This was fixed with following pr - #511